builder/frameworks/_embed_files.py

```
@@ -24,6 +25,14 @@
 mcu = board.get("build.mcu", "esp32")
 is_xtensa = mcu in ("esp32", "esp32s2", "esp32s3")
 
+cmake_dir = env.PioPlatform().get_package_dir("tool-cmake")
```

Ruff (0.12.2)

Undefined name `env`

(F821)